plugins.d/alarm-notify.sh

```diff
@@ -795,6 +852,17 @@ The source of this alarm is line ${src}"
 
 SENT_PUSHBULLET=$?
 
+# send the twilio SMS
+
+send_twilio "${TWILIO_ACCOUNT_SID}" "${TWILIO_ACCOUNT_TOKEN}" "${TWILIO_NUMBER}" "${to_twilio}" "${host} ${status_message} - ${name//_/ } - ${chart}" "${alarm} 
+Severity: ${severity}
+Chart: ${chart}
+Family: ${family}
+To View Netdata go to: ${goto_url}
+The source of this alarm is line ${src}"
```

Comment:
I can do that, but so far each alarm has come in as one msg. I didn't like having the URL in the msg any ways, lots of clutter and I don't expect much people to actually use netdata on the phone unless it's an emergency and they know how to reach the dashboard.